subprojects/docs/src/docs/userguide/what_is_gradle.adoc

```diff
@@ -12,133 +12,114 @@
+Several major IDEs provide interaction with Gradle builds, including Android Studio, IntelliJ IDEA, Eclipse, VSCode, and NetBeans.
+Gradle can also generate the solution files required to load a project into Visual Studio.
 Insight::
-https://scans.gradle.com/[Build scans] provide extensive information about a build run that you can use to identify build issues. They are particularly good at helping you to identify problems with a build's performance. You can also share build scans with others, which is particularly useful if you need to ask for advice in fixing an issue with the build.
-
-[[five_things]]
-== Five things you need to know about Gradle
-
-Gradle is a flexible and powerful build tool that can easily feel intimidating when you first start. However, understanding the following core principles will make Gradle much more approachable and you will become adept with the tool before you know it.
+https://scans.gradle.com/[Build scans] provide extensive information about a build that you can use to identify issues.
+You can use build scans to identify problems with a build's performance, and even share them for debugging help.
 
-=== 1. Gradle is a general-purpose build tool
+== Concepts
 
-Gradle allows you to build any software, because it makes few assumptions about what you're trying to build or how it should be done. The most notable restriction is that dependency management currently only supports Maven- and Ivy-compatible repositories and the filesystem.
+It's helpful to know the following concepts before you dive into the details of Gradle.
 
-This doesn't mean you have to do a lot of work to create a build. Gradle makes it easy to build common types of project — say Java libraries — by adding a layer of conventions and prebuilt functionality through <<plugins#plugins,_plugins_>>. You can even create and publish custom plugins to encapsulate your own conventions and build functionality.
+=== Projects
 
-[[the_core_model_is_based_on_tasks]]
-=== 2. The core model is based on tasks
+Projects are the things that Gradle builds.
+Projects contain a build script, which is a file located in the project's root directory named `build.gradle` or `build.gradle.kts`.
+Builds scripts define tasks, dependencies, plugins, and other configuration for your build.
+A single project can contain many subprojects, which can each contain their own subprojects.
 
-Gradle models its builds as Directed Acyclic Graphs (DAGs) of tasks (units of work). What this means is that a build essentially configures a set of tasks and wires them together — based on their dependencies — to create that DAG. Once the task graph has been created, Gradle determines which tasks need to be run in which order and then proceeds to execute them.
+=== Tasks
```

Comment:
I think I would put this section below Plugins

Reply:
I intentionally placed this section before plugins because IMO tasks are a building block of plugins. But others have raised the point that many introductory users don't write tasks at all -- they just use plugins. So maybe we should put plugins first?

I'm concerned that the concept of plugins is pretty abstract unless you anchor it in something more concrete like tasks. I'll defer to your opinion on this.

Reply:
I agree with Nate here.  You almost always use a plugin to define tasks, but you can define tasks outside of plugins.  The tasks section seems like it is independent of plugins and should come first.